app/connectors/httpParsers/SubmittedDividendsHttpParser.scala

```diff
@@ -41,7 +42,11 @@ object SubmittedDividendsHttpParser extends DESParser {
         case SERVICE_UNAVAILABLE =>
           pagerDutyLog(SERVICE_UNAVAILABLE_FROM_DES, logMessage(response))
           handleDESError(response)
-        case BAD_REQUEST | NOT_FOUND =>
+        case NOT_FOUND =>
+          logger.info(logMessage(response))
+          pagerDutyLog(FOURXX_RESPONSE_FROM_DES, logMessage(response))
```

Comment:
pagerDutyLog(FOURXX_RESPONSE_FROM_DES, logMessage(response)) should be removed

Reply:
Removed